sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -181,7 +190,7 @@ export abstract class LinkEntity<LinkT extends Receiver | AwaitableSender | Requ
     this.address = options.address || "";
     this.audience = options.audience || "";
     this.name = getUniqueName(name);
-    this._logPrefix = `[${context.connectionId}|${this._linkType}:${this.name}|a:${this.address}]`;
```

Comment:
The address has been helpful in logs when the user is working with multiple queues/subscriptions. What is the reason to remove it?

Reply:
Going through some of the logging examples it looks like the 'address' is more or less redundant with the link name (which already encodes the entity name).